docs/standard/microservices-architecture/architect-microservice-container-applications/microservices-architecture.md

```diff
@@ -43,10 +43,10 @@ Of these, only the first three are covered or introduced in this guide. The last
   [*https://azure.microsoft.com/blog/microservices-an-application-revolution-powered-by-the-cloud/*](https://azure.microsoft.com/blog/microservices-an-application-revolution-powered-by-the-cloud/)
 
 - **Martin Fowler. Microservices** \
-  [*http://www.martinfowler.com/articles/microservices.html*](http://www.martinfowler.com/articles/microservices.html)
+  [*https://www.martinfowler.com/articles/microservices.html*](https://www.martinfowler.com/articles/microservices.html)
```

Comment:
There's the same problem with this link and the link on line 49: the link text continues to be rendered as http. The same fix from above to unify link text and link might work.